docs: add jq doc for editing genesis file (#9918)
cosmos/cosmos-sdk: docs/run-node/run-node.md

      |- node_key.json              # Private key to use for node authentication in the p2p protocol.
      |- priv_validator_key.json    # Private key to use as a validator in the consensus protocol.
```

> If you want change any field values in the json files(ex: genesis.json) you can use `jq` ([installation](https://stedolan.github.io/jq/download/) & [docs](https://stedolan.github.io/jq/manual/#Assignment)) command to do that (for ex: ```jq '.chain_id = "testing1"' genesis.json > temp.json && mv temp.json genesis.json```)

Reviewer: ```suggestion
If you want change any field values in the json files (for ex: genesis.json) you can use `jq` ([installation](https://stedolan.github.io/jq/download/) & [docs](https://stedolan.github.io/jq/manual/#Assignment)) command to do that. For example, to change the genesis chain ID:
```bash
jq '.chain_id = "testing1"' genesis.json > temp.json && mv temp.json genesis.json
```
```

Reviewer 2: I believe this should be a new section for better visibility. Eg `## Updating genesis fields`

> If you want change any field values in the json files(ex: genesis.json) you can use `jq` ([installation](https://stedolan.github.io/jq/download/) & [docs](https://stedolan.github.io/jq/manual/#Assignment)) command to do that (for ex: ```jq '.chain_id = "testing1"' genesis.json > temp.json && mv temp.json genesis.json```)

Reviewer 3: ```suggestion
If you want to change any field values in the json files (ex: genesis.json) you can use `jq` ([installation](https://stedolan.github.io/jq/download/) & [docs](https://stedolan.github.io/jq/manual/#Assignment)) command to do that (for ex: ```jq '.chain_id = "testing1"' genesis.json > temp.json && mv temp.json genesis.json```)
```

      |- priv_validator_key.json    # Private key to use as a validator in the consensus protocol.
```

## Updating Some Default Settings

If you want to change any field values in configuration files (for ex: genesis.json) you can use `jq` ([installation](https://stedolan.github.io/jq/download/) & [docs](https://stedolan.github.io/jq/manual/#Assignment)) & `sed` commands to do that. Few examples are listed here.
```bash
# to change the chain-id
jq '.chain_id = "testing"' genesis.json > temp.json && mv temp.json genesis.json

# to enable the api server
sed -i '/\[api\]/,+3 s/enable = false/enable = true/' app.toml

# to change the voting_period
jq '.app_state.gov.voting_params.voting_period = "600s"' genesis.json > temp.json && mv temp.json genesis.json

# to change the inflation
jq '.app_state.mint.minter.inflation = "0.300000000000000000"' genesis.json > temp.json && mv temp.json genesis.json
```

> 

Reviewer: ```suggestion

## Adding Genesis Accounts
```

propose to add this subheader, because the next paragraphs talk about something else
